<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="WebController" name="web" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStation">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionStation_Window"/>
    <addaction name="actionDVR_Window"/>
    <addaction name="actionTele_Window"/>
   </widget>
   <addaction name="menuStation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStation_Window">
   <property name="text">
    <string>Station Window</string>
   </property>
  </action>
  <action name="actionDVR_Window">
   <property name="text">
    <string>DVR Window</string>
   </property>
  </action>
  <action name="actionTele_Window">
   <property name="text">
    <string>Tele Window</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebController</class>
   <extends>QWidget</extends>
   <header>webcontroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
